### PR TITLE
chore: update to storybook v6.4.9

### PR DESCRIPTION
**Pull request by nataliepina**
Storybook suggested installing `eslint-plugin-storybook` for the best experience, which in turn updated the .eslintrc file. 

## Testing

Test out starting storybook and running npm install. 

## Screenshots
Screen Shot 2022-01-07 at 8 24 39 AM

## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [ ] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [x] Info for applicable sections above is provided

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -112,6 +112,7 @@
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-react-hooks": "^4.3.0",
+    "eslint-plugin-storybook": "^0.5.5",
```
Comment (clintonmedbery) [inline package.json:115]: Do we want the ^

Comment (nataliepina) [reply package.json:115]: Yes, during the version update storybook suggested something along the lines of "for the best storybook experience we now highly suggest also installing eslint-plugin-storybook" so I went with that which in turn asked to update the .eslintrc file.

Comment (nataliepina) [reply package.json:115]: Screen Shot 2022-01-07 at 8 24 39 AM

Comment (clintonmedbery) [reply package.json:115]: I meant the actual caret ^

**Review by clintonmedbery — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (nataliepina) [reply package.json]: Oops. Fixed!

**Pull request merged**